src/main/java/io/jenkins/plugins/analysis/core/model/BlamesModel.java

```diff
@@ -47,12 +48,25 @@
                 Messages.Table_Column_Age(),
                 Messages.Table_Column_Author(),
                 Messages.Table_Column_Email(),
-                Messages.Table_Column_Commit());
+                Messages.Table_Column_Commit(),
+                Messages.Table_Column_AddedAt());
+    }
+
+    @Override
+    public List<String> getHeaderClasses(final Report report) {
```

Comment:
I wonder if it would make more sense if I replace the `n` column methods with a single method that returns a list of objects. What do you think?

Reply:
Yes, I think that'd be better! It would simplify the creation of the table head in the `table.jelly` file. And another big advantage would be that the sizes of the lists can no longer differ from each other!

Should I change this directly in this PR?

Reply:
I think you can leave your code as it is. If the unit test will pass I integrate them and merge the results with https://github.com/jenkinsci/warnings-ng-plugin/pull/252 later on on my self. Otherwise the rework will be too much for you (and me) since the API for the JS access (tables and so on) has been changed quite a lot due to the extraction of the new JS libs.